docs/Prodigy_training_data.md

```diff
@@ -79,63 +79,47 @@ python nutrition_labels/prodigy_training_data.py --prodigy_data_dir 'data/prodig
 ```
 which will output the training data in a datestamped folder.
 
-This outputted the `data/processed/training_data/210218/training_data.csv` training dataset:
+This outputted the `data/processed/training_data/210221/training_data.csv` training dataset:
 
 | Tag code | Meaning | Number of grants |
 |---|---|--- |
-| 1 | Relevant | 592 |
-| 0 | Not relevant | 576 |
+| 1 | Relevant | 495 |
+| 0 | Not relevant | 485 |
 
 ### Variability in the results:
 
-I reran `grant_tagger_seed_experiments.py` with the training data in `data/processed/training_data/210218/training_data.csv`. This script trains several models with different random seeds and teamed with the `Seed variability.ipynb` notebook we can pick a new 'best' random seed to choose our training/test split for all models.
+
+I reran `grant_tagger_seed_experiments.py` with the training data in `data/processed/training_data/210221/training_data.csv`. This script trains several models with different random seeds and teamed with the `Seed variability.ipynb` notebook we can pick a new 'best' random seed to choose our training/test split for all models.
```

Comment:
It is worth pausing and thinking whether picking the best seed is a good idea. I think researchers do that when they publish results but ideally what you want is low variance to begin with and any model not high variance and the best model there as this overfits the validation/test data.

I would like to hear @aCampello thoughts here.

Reply:
I haven't followed this very closely, so I am not sure what the cross validation technique is, but I suspect that if a `n-fold` (say 5-fold) validation is used, the effect of the seed will be smoothed. I agree that people use it to guarantee that your results are replicable, not intended as a hyper parameter.

From what I saw from the notebook, the effect of the random seed is like 1%-ish, which is probably ok?

Reply:
yes a good point about overfitting to the test data, especially since the dataset is so small too.